gui/src/components/Layout.tsx

```diff
@@ -197,14 +203,16 @@ const Layout = () => {
 
   // Check if there are any incompatible extension enabled when the webview is on mount
   useEffect(() => {
-    ideMessenger.post("checkForIncompatibleExtensions", undefined)
+    ideMessenger.post("checkForIncompatibleExtensions", undefined);
   }, []);
 
   return showGraniteOnboardingCard ? (
     <GraniteOnboardingCard />
   ) : (
     <>
-    {conflictsWithOtherExtensions && <IncompatibleExtensionsOverlay />}
+    {conflictsWithOtherExtensions && conflictingInfo !== null && (
```

Comment:
you can remove conflictsWithOtherExtensions, it's redundant with conflictingInfo